app/models/missing_model.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+class missing_model extends Model {
+
+    private function execute($sql, $datatype = null, $data = array()) {
+        $stmt = $this->DB->prepare($sql);
+        if (!$stmt) {
+            return false;
+        }
+        if (!empty($datatype) && !empty($data)) {
+            if (!call_user_func_array(array($stmt, "bind_param"), array_merge([$datatype], $data))) {
+                return false;
+            }
+        }
+        if (!$stmt->execute()) {
+            return false;
+        }
+        $result = $stmt->get_result();
```

Comment:
You must use the newer [`mysqlnd`](http://php.net/manual/en/book.mysqlnd.php) driver with PHP. Sorry, I should have documented this before. I'll update the README soon.